panel/tests/widgets/test_remote_file_selector.py

```diff
@@ -0,0 +1,20 @@
+# to fake delays in DummyRemoteFileProvider
+import time
+
+from pathlib import PurePosixPath
+
+from panel.widgets import RemoteFileProvider
+
+
+class DummyRemoteFileProvider(RemoteFileProvider):
+
+    def __init__(self):
+        super().__init__()
+
+    async def ls(self, path:PurePosixPath):
+        time.sleep(1)
```

Comment:
```suggestion
        asyncio.sleep(1)
```

So it makes sense with async.